cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/AggregatingFunctionsTest.scala

```diff
@@ -29,121 +29,129 @@ class AggregatingFunctionsTest extends DocumentingTest {
+        ("expression", "An expression returning a set of values."))
+      considerations(
+        "Any `null` values are ignored and will not be added to the list.",
+        "`collect(null)` returns an empty list.")
+      query("""MATCH (n:Person)
+              #RETURN collect(n.age)""".stripMargin('#'),
+      ResultAssertions((r) => {
         r.toList.head("collect(n.age)") should equal(Seq(13, 33, 44))
       })) {
         p("All the values are collected and returned in a single list.")
         resultTable()
       }
     }
     section("count()", "functions-count") {
-      p(
-        """`count()` returns the number of values or rows, and appears in two variants:
-          |
-          |* `count(*)` returns the number of matching rows, and
-          |* `count(expr)` returns the number of non-`null` values returned by an expression.
-        """.stripMargin)
-      function("count(expression)", "An Integer.", ("expression", "An expression."))
-      considerations("`count(*)` includes rows returning `null`.", "`count(expr)` ignores `null` values.", "`count(null)` returns `0`.")
+      p("""The function `count()` returns the number of values or rows, and appears in two variants:
+          #
+          #`count(*)`:: returns the number of matching rows.
+          #`count(expr)`:: returns the number of non-`null` values returned by an expression.""".stripMargin('#'))
+      function("count(expression)",
+        "An Integer.",
+        ("expression", "An expression."))
+      considerations(
+        "`count(*)` includes rows returning `null`.",
+        "`count(expr)` ignores `null` values.",
+        "`count(null)` returns `0`.")
       section("Using `count(*)` to return the number of nodes") {
-        p("`count(*)` can be used to return the number of nodes; for example, the number of nodes connected to some node `n`.")
-        query("MATCH (n {name: 'A'})-->(x) RETURN labels(n), n.age, count(*)", ResultAssertions((r) => {
+        p("The function `count(*)` can be used to return the number of nodes; for example, the number of nodes connected to some node `n`.")
+        query("""MATCH (n {name: 'A'})-->(x)
+                #RETURN labels(n), n.age, count(*)""".stripMargin('#'),
+        ResultAssertions((r) => {
           r.toList should equal(List(Map("labels(n)" -> List("Person"), "n.age" -> 13L, "count(*)" -> 3L)))
         })) {
           p("The labels and `age` property of the start node `n` and the number of nodes related to `n` are returned.")
           resultTable()
         }
       }
       section("Using `count(*)` to group and count relationship types") {
-        p("`count(*)` can be used to group relationship types and return the number.")
-        query("MATCH (n {name: 'A'})-[r]->() RETURN type(r), count(*)", ResultAssertions((r) => {
+        p("The function `count(*)` can be used to group relationship types and return the number.")
```

Comment:
The number of types or relationships?

Reply:
I have updated the example to include a new relationship READS and rephrase the text. It is grouping the type (only relationships has at max one type, nodes has labels)